qownnotes/snapcraft.yaml

```diff
@@ -10,7 +10,7 @@ description: |
 
 apps:
   qownnotes:
-    command: desktop-launch usr/bin/QOwnNotes --snap
+    command: desktop-launch ${SNAP}/usr/bin/QOwnNotes --snap
```

Comment:
That's due to usr/bin/ relative path.
However, `desktop-launch QOwnNotes` should work (as $SNAP/usr/bin is automatically added to $PATH)